### PR TITLE
Allow exit code 1 from grep in make linkcheckbroken

### LINKED ISSUE #1212: linkcheckbroken is broken
### Plone Documentation versions affected

- [ ] 5.x
- [X] 6.x

### Description

`make linkcheckbroken` returns an unhelpful error message:

`make: *** [linkcheckbroken] Error 1`

However:

`make linkcheck | GREP_COLORS='0;31' grep -wi "broken\|redirect" --color=auto`

Works just fine. This is identical to the former in `Makefile`.

The only thing that might be causing the error is possibly the check for `html_meta` values returning a non-zero exit code.

This issue affects `plone/volto` and `plone/plone.restapi` as well, except for prefixing the commands with `docs-`.

For now we have changed the CI to use `make linkcheck` instead of `make linkcheckbroken` across all the repos.

### PR DESCRIPTION
**Pull request by wolbernd**
Fixes #1212

### CODE REVIEW

**Review by stevepiercy — CHANGES_REQUESTED** [review]
One more change for tests:

```
test: clean linkcheckbroken spellcheck  ## Clean docs build, then run linkcheckbroken, spellcheck
```

That will vastly improve the noisy output from `linkcheck`.

File: Makefile [hunk 1]
```diff
@@ -212,7 +212,7 @@ doctest: deps
 	      "results in $(BUILDDIR)/doctest/output.txt."
 
 .PHONY: test
-test: clean linkcheck spellcheck  ## Clean docs build, then run linkcheck, spellcheck
+test: clean linkcheckbroken spellcheck  ## Clean docs build, then run linkcheck, spellcheck
```
Comment (stevepiercy) [inline Makefile]:
```suggestion
test: clean linkcheckbroken spellcheck  ## Clean docs build, then run linkcheckbroken, spellcheck
```

**Comment by wolbernd** [comment]
@stevepiercy `make test` has been updated

**Review by stevepiercy — CHANGES_REQUESTED** [review]
Just one nitpick

**Review by stevepiercy — APPROVED** [review]
YAY!

**Comment by stevepiercy** [comment]
Look at that! 🥳 

https://github.com/plone/documentation/runs/6349601870?check_suite_focus=true#step:5:867

**Pull request merged**